packages/prerender-data-provider/package.json

```diff
@@ -1,29 +1,24 @@
 {
   "name": "@preact/prerender-data-provider",
   "version": "3.0.1",
-  "description": "Prerender Prerender data provider for preact-cli apps",
+  "description": "Prerender data provider for preact-cli apps",
   "keywords": [
     "preact",
     "preact-cli"
   ],
-  "author": "Preact Authors <core@preactjs.com>",
-  "homepage": "https://github.com/developit/preact-cli#readme",
-  "license": "MIT",
-  "main": "src/index.js",
-  "directories": {
-    "lib": "src",
-    "test": "__tests__"
-  },
-  "files": [
-    "src"
-  ],
+  "homepage": "https://github.com/preactjs/preact-cli",
+  "bugs": "https://github.com/preactjs/preact-cli/issues",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/developit/preact-cli.git"
-  },
-  "bugs": {
-    "url": "https://github.com/developit/preact-cli/issues"
+    "url": "https://github.com/preactjs/preact-cli.git",
+    "directory": "packages/prerender-data-provider"
   },
+  "license": "MIT",
+  "author": "The Preact Authors (https://preactjs.com)",
+  "main": "lib/index.js",
```

Comment:
I personally really dislike `lib` in the root dir, would kinda prefer moving the CLI code to src